File: app/src/main/java/in/completecourse/model/NotificationModel.java
package in.completecourse.model;

public class NotificationModel {
    private String mHeading;
    private String mSubHeading;
    private String url;
    private String serial;

    public String getmHeading() {
        return mHeading;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getmSubHeading() {
        return mSubHeading;
    }

    public void setmHeading(String mHeading) {
        this.mHeading = mHeading;
    }

    public void setmSubHeading(String mSubHeading) {
        this.mSubHeading = mSubHeading;
    }

    public NotificationModel(){

    }

    public String getSerial() {
        return serial;
    }

    public void setSerial(String serial) {
        this.serial = serial;
    }

    public NotificationModel(String heading, String subheading, String url){
        this.mHeading = heading;
        this.mSubHeading = subheading;
        this.url = url;
    }
}

File: app/src/main/java/in/completecourse/model/NotificationModel.kt
package `in`.completecourse.model

class NotificationModel {
    private var mHeading: String? = null
    private var mSubHeading: String? = null
    var url: String? = null
    var serial: String? = null
    fun getmHeading(): String? {
        return mHeading
    }

    fun getmSubHeading(): String? {
        return mSubHeading
    }

    fun setmHeading(mHeading: String?) {
        this.mHeading = mHeading
    }

    fun setmSubHeading(mSubHeading: String?) {
        this.mSubHeading = mSubHeading
    }

    constructor() {}

    constructor(heading: String?, subheading: String?, url: String?) {
        mHeading = heading
        mSubHeading = subheading
        this.url = url
    }
}